src/core/or/circuitpadding.c

```diff
@@ -851,12 +896,59 @@ circpad_machine_remove_token(circpad_machine_runtime_t *mi)
     mi->nonpadding_sent /= 2;
   }
 
+  /* Update any state packet length limits that apply */
+  circpad_machine_update_state_length(mi);
+
+  /* Remove a token from the histrogram, if applicable */
+  circpad_machine_remove_token(mi);
+}
+
+static inline void
```

Comment:
Let's document this.

Reply:
This was done in the new PR, #1004 (see 8cfd329).